### DISCUSSION TITLE
Troubleshooting SLTS 2.7.5.3 installer on Debian 12

### DISCUSSION THREAD

**Discussion by fabiodurao** [SCADA-LTS/Scada-LTS#2605] [General]
I'm installing SLTS 2.7.5.3 on Debian 12 64bit and got the following error.

```
MySQL Shell version 8.0.33 installed
./mysqld: error loading shared libraries: libnuma.so.1: cannot open shared object file: no such file or directory
MySQL Community Server version 8.0.33 configured
./mysqld: error loading shared libraries: libnuma.so.1: cannot open shared object file: no such file or directory
```

I solved it by installing the following packages and running the script again:

`sudo apt-get install libnuma-dev
`

![image](https://github.com/SCADA-LTS/Scada-LTS/assets/38673539/7c3d975f-a935-427a-bf13-aa6e3f0ec316)

**Comment by fabiodurao** [comment]
I had one more problem in the installation, I used the following password for the msql user
"CK9y#j&H%mWm"

in the file ~/tomcat64/server/webapps/Scada-LTS/META-IN/context.xml 2 passwords appeared, from what I understand, as there was an "&" in the middle of the password, it was treated with 2 strings, see how the password inside the <resource> tag:

           username="scadalts"
           password="CK9y#jpassword=""H%mWm"
           driverClassName="com.mysql.jdbc.Driver"

**Comment by Limraj** [comment]
Hi @fabiodurao,
thanks for these comments:

1. We currently support distributions from the ubuntu family (ubuntu/mint), we are unlikely to be able to guarantee that it will work on every distribution. If support for Debian is crucial, we can add this extra step for such a system, i.e. installing this package.

2. As for the password, I'll check what's going on and try to improve it.

Regards,
Kamil Jarmusik

**Comment by fabiodurao** [comment]
Thank you @Limraj for this information,

I'm going to migrate to Ubuntu to stay within what is supported, for now I don't see the need to focus on Debian.
